server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -571,6 +602,19 @@ const orderMutations = {
               },
             },
           );
+
+          // Update paymentMethodId in transactions for Added Funds
+          if (addedFundOrders.length > 0 && isCollective) {
+            await models.Transaction.update(
+              { PaymentMethodId: addedFundPaymentMethod.id },
+              {
+                transaction: dbTransaction,
+                where: {
+                  [Op.or]: addedFundOrders.map(order => ({ OrderId: order.id })),
+                },
+              },
+            );
+          }
```

Comment:
We already have some `// Update transactions` above, why not just add `PaymentMethodId` there?

Reply:
This seems a bit complicated or maybe I am missing something simple. Note that the were clause is different here as we need to only consider added fund orders. Plus the fields to update is also different. I thought trying to do this in the above transactions makes it too complicated with some conditionals. 🤔